src/content/concepts/index.md

```diff
@@ -154,3 +151,16 @@ There are many plugins that webpack provides out of the box! Check out our [list
 Using plugins in your webpack config is straightforward - however, there are many use cases that are worth further exploration.
 
 [Learn more!](/concepts/plugins)
+
+
+## Mode
+
+By choosing between `development` and `production` **mode** you can enable webpack's automatic optimizations according to selected option.
+
+```javascript
+module.exports = {
+  mode: 'production'
+};
+```
+
+[Learn more!](/concepts/mode)
```

Comment:
I don't think this page exists (yet), so better take this out for now :)

Reply:
But i did add it within the pr

Reply:
Oh yeah, completely missed that because of the branch switch, sorry about that!